### ISSUE TITLE
[DESKTOP - Home Page] Categories

### ISSUE THREAD

**Issue by NeOMakinG** [PrestaShop/hummingbird#40]
Epic: https://github.com/PrestaShop/theme-refacto/issues/2
Responsive's issue: https://github.com/PrestaShop/theme-refacto/issues/39

[Link to Figma prototype](https://www.figma.com/file/LfVl5leeSKcVUhSaYwhbtM/New-Theme?node-id=47%3A286)

#### UX and design explanations
- The category section allows users to access a specific category
- It also allows merchants to showcase some of their categories
- This section is positioned between the banner and the popular products

#### Mockup
![desktop - categories](https://user-images.githubusercontent.com/54888056/135830411-d9efb51b-a8e3-4bbd-9fa8-71c6aa7fad6a.png)

**Comment by Hlavtox** [comment]
@NeOMakinG I have this coded in one of my client's shops using display: grid. We can take inspiration. See https://eshop.agrola.cz/

Two things:
- What if we made this universal, so it's not limited to categories and anybody could put any links in there?
- Can we make it not limited to 3 grid layout, so anybody can use as many items as they want? So we can have more rich grids like here? https://www.cycli.cz/
- We will need to create a new module for this. Can you create a repo for it? Should be called something like ps_homebanner or something like this.

**Comment by SharakPL** [comment]
@Hlavtox I see you went with background and cover in your project. IMO not the best choice for category images. To improve SEO I would use &lt;img&gt; and `object-fit: cover`.
Besides this is tricky to implement this way (1 / 2). For example if there's 4 categories it would have to be 2 / 2 or 1 big / 3 small. This would require all categories to have big thumbnails generated. Since this is just default theme then maybe it would be best to make it simple with all categories equal in size. Unless it's going to be something like themeconfigurator module in PS 1.6 where you would upload custom images for each item, without using generated thumbnails.

**Comment by Hlavtox** [comment]
@SharakPL Yes, it would be better to use the img tag for better control. ;) Honestly, I learned about object-fit not long ago, thats why I didnt use it.

I would let the user to upload a custom photo and set a custom link for each of the blocks, so they can put a link to CMS or whatever they want.

**Comment by NeOMakinG** [comment]
@PrestaShop/product-team, This thing is a complete new part, should it be a module? Like the slider? 

@Hlavtox @SharakPL, wdyt?

**Comment by SharakPL** [comment]
IMO category images should be generated like now. One improvement is required though: if there is a category miniature set, then changing main category image shouldn't change the miniature. It should only generated miniature from main image if there's no miniature specified.

Adding custom image/text/link/html can be placed on any hook, should be in separate module.

**Comment by Hlavtox** [comment]
@ga-devfront Why are you closing everything we agreed on???
